components/patient-data/ui/src/main/resources/PhenoTips/PatientSheetCode.xml

```diff
@@ -1341,9 +1341,9 @@ document.observe('xwiki:dom:loaded', function()
             return;
           }
           var ynpickerElt = YesNoPicker.generatePickerElement([
-              {type: 'na', selected: !isValueSelected(positiveName, value) &amp;&amp; !isValueSelected(negativeName, value)},
-              {type: 'yes', selected: isValueSelected(positiveName, value)},
-              {type: 'no' , selected: isValueSelected(negativeName, value)}
+              {type: 'na' , name: positiveName + '_na', selected: !isValueSelected(positiveName, value) &amp;&amp; !isValueSelected(negativeName, value)},
+              {type: 'yes', name: positiveName, selected: isValueSelected(positiveName, value)},
```

Comment:
Hm, the same thing will now be sent multiple times to the server, and this may mess up the value. Is there something that de-duplicates the values?